python/ray/tune/trainable/trainable.py

```diff
@@ -1146,23 +1147,34 @@ def load_checkpoint(self, checkpoint: Union[Dict, str]):
         The directory structure under the checkpoint_dir provided to
         ``Trainable.save_checkpoint`` is preserved.
 
-        See the example below.
+        See the examples below.
 
         Example:
             >>> from ray.tune.trainable import Trainable
             >>> class Example(Trainable):
             ...    def save_checkpoint(self, checkpoint_path):
-            ...        print(checkpoint_path)
-            ...        return os.path.join(checkpoint_path, "my/check/point")
-            ...    def load_checkpoint(self, checkpoint):
-            ...        print(checkpoint)
+            ...        my_checkpoint_path = os.path.join(checkpoint_path, "my/path")
```

Comment:
We're still skipping most `doctests`, but I think here we should at least make sure to have `os`.